wishlist/app/com/commercetools/sunrise/wishlist/WishlistFinderBySession.java

```diff
@@ -0,0 +1,61 @@
+package com.commercetools.sunrise.wishlist;
+
+import com.commercetools.sunrise.framework.hooks.HookRunner;
+import com.commercetools.sunrise.sessions.customer.CustomerInSession;
+import com.commercetools.sunrise.sessions.wishlist.WishlistInSession;
+import com.commercetools.sunrise.wishlist.controllers.AbstractSingleShoppingListQueryExecutor;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.shoppinglists.ShoppingList;
+import io.sphere.sdk.shoppinglists.queries.ShoppingListQuery;
+
+import javax.inject.Inject;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+
+public class WishlistFinderBySession extends AbstractSingleShoppingListQueryExecutor implements WishlistFinder {
```

Comment:
This class doesn't provide any methods to override (only get, which is the same as implementing your own class from scratch) so I would make it package scope and final.